manifests/b/Baidu/BaiduPinyin/5.6.5624.0/Baidu.BaiduPinyin.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Baidu.BaiduPinyin
+PackageVersion: 5.6.5624.0
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- bdict
+- bps
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: neutral
```

```suggestion
- Architecture: neutral
```
It looks like we need to start to move away from using `InstallerLocale` here because this prevents machines outside of the `zh-CN` locale from upgrading the application using the `winget upgrade` command.

See the comment that @denelon wrote at https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903016232 & https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903033816.